lib/fastlane/plugin/sentry/actions/sentry_upload_dif.rb

```diff
@@ -53,15 +52,27 @@ def self.details
       def self.available_options
         Helper::SentryConfig.common_api_config_items + [
           FastlaneCore::ConfigItem.new(key: :path,
-                                       description: "A path to search recursively for symbol files",
+                                       env_name: "SENTRY_DSYM_PATH",
+                                       description: "Path to search recursively for symbol files",
+                                       default_value: Actions.lane_context[SharedValues::DSYM_OUTPUT_PATH],
+                                       is_string: false,
```

Comment:
Think we don't need the `is_string: false` param here. AFAIK it will only check the string type if set to true, and it is deprecated. https://github.com/fastlane/fastlane/blob/258cddbe2c1f04378d89ef67d661a929bbc54414/fastlane_core/lib/fastlane_core/configuration/config_item.rb#L85

Reply:
right - but is_string defaults to `true`, which means if we drop `is_string: false`, we'll need to replace it with `type:`, otherwise the type will revert to `String`.  The documentation isn't very clear in what the value for `type:` should be.